tests/tests/query/builder/InsertTest.php

```diff
@@ -10,6 +10,12 @@
  */
 class Jam_Query_Builder_InsertTest extends PHPUnit_Framework_DOMTestCase {
 
+	public function tearDown()
+	{
+		Jam::clear_cache();
```

Comment:
Well I'm not sure how the tests were passing before without it. It started failing after I've updated the everything and this fixes it. The insert query builder test is the only test of query builder tests in Jam that tests the events. When the cache of models is not cleared there's shared state between the tests inside of the Jam meta and event callbacks are called more than once and the mock expectations are failing.